.gitpod.Dockerfile

```diff
@@ -1,13 +1,19 @@
-FROM gitpod/workspace-full:2023-05-08-21-16-55
+# FROM gitpod/workspace-full:2023-05-08-21-16-55
 
 # Some datasets work on 3.8 only
-RUN pyenv install 3.8.15\
-    && pyenv global 3.8.15
+# RUN pyenv install 3.8.15\
+#     && pyenv global 3.8.15
+
+# You could use `gitpod/workspace-full` as well.
+FROM gitpod/workspace-python
+
+RUN pyenv install 3.11.1 \
+    && pyenv global 3.11.1
 
 # VideoDataSet
-RUN sudo apt-get update && sudo apt-get install -y --no-install-recommends libgl1
+# RUN sudo apt-get update && sudo apt-get install -y --no-install-recommends libgl1
 RUN sudo apt-get install make
-RUN npm install -g @mermaid-js/mermaid-cli
+# RUN npm install -g @mermaid-js/mermaid-cli
 # https://stackoverflow.com/questions/69564238/puppeteer-error-failed-to-launch-the-browser-process
 # https://github.com/puppeteer/puppeteer/blob/main/docs/troubleshooting.md#chrome-doesnt-launch-on-linux
-RUN sudo apt-get install -y --no-install-recommends libatk-bridge2.0-0 libcups2 ca-certificates fonts-liberation libasound2 libatk-bridge2.0-0 libatk1.0-0 libc6 libcairo2 libcups2 libdbus-1-3 libexpat1 libfontconfig1 libgbm1 libgcc1 libglib2.0-0 libgtk-3-0 libnspr4 libnss3 libpango-1.0-0 libpangocairo-1.0-0 libstdc++6 libx11-6 libx11-xcb1 libxcb1 libxcomposite1 libxcursor1 libxdamage1 libxext6 libxfixes3 libxi6 libxrandr2 libxrender1 libxss1 libxtst6 lsb-release wget xdg-utils
+# RUN sudo apt-get install -y --no-install-recommends libatk-bridge2.0-0 libcups2 ca-certificates fonts-liberation libasound2 libatk-bridge2.0-0 libatk1.0-0 libc6 libcairo2 libcups2 libdbus-1-3 libexpat1 libfontconfig1 libgbm1 libgcc1 libglib2.0-0 libgtk-3-0 libnspr4 libnss3 libpango-1.0-0 libpangocairo-1.0-0 libstdc++6 libx11-6 libx11-xcb1 libxcb1 libxcomposite1 libxcursor1 libxdamage1 libxext6 libxfixes3 libxi6 libxrandr2 libxrender1 libxss1 libxtst6 lsb-release wget xdg-utils
```

Comment:
It's temporary change so I can focus debugging in Python 3.11, I will revert the changes when it is done.